### ISSUE TITLE
Getting Page/Post Slug

### ISSUE THREAD

**Issue by sakinshrestha** [lightspeeddevelopment/lsx#155]
* In `inc/extras.php`, lines 26-29, you have used method to add page/post slug to body class
* Use following method instead:
```
    global $post;
    if ( isset( $post ) ) {
        $classes[] = $post->post_name;
    }
```

**Comment by justintadlock** [comment]
I'd actually use this since it's outside of The Loop:

```
get_queried_object()->post_name
```

I try to avoid global `$post` when possible in favor of a core function.

**Comment by nandotess** [comment]
Resolved on LSX 2.0.0 branch.
